### ISSUE TITLE
MeditationTimerSVC

### ISSUE THREAD

**Issue by DrexelOMS** [DrexelOMS/OvercomingMS#141]
theres a few classes in the timer area, main timer code, the two abstract sub classes for stopwatch, and countdown timer. 

Meditation is obviously the latter and would need a way to set the initial time. i.e. set the second an than start and stop the timer, and do something once the timer is done, or still be able to correctly calculate the time from a start time - seconds if they clicked finish early

it would normally get this start time from a setup page, but for now  you can hardcode a start time of 15 minutes, and we will make the setup pages in a separate issue

Implementation Notes:
I have already setup the 2 setup views, and they send along the start time in Seconds to the MeditationTimerSVC. this class descends from the TimerAbstract class, which currently just switches the update from adding seconds to removing seconds, there should be a check to pause the timer if seconds is zero. Finally if you look at the exercise class, make sure to update the on finish pressed method to correctly return to the main page after it adds their meditation time with the length of time they spent meditating, I have code in exercise that handles rounding up to the next minute i believe?
